app/views/content/non-uk-teachers/find-a-teaching-job-as-a-non-uk-citizen.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Find a teaching job as a non-UK citizen"
+subcategory: If you want to train to teach
+description: |-
+  Learn more about applying for teaching jobs as a non-UK citizen after you qualify as a teacher.
+related_content:
+    Train to teach in England as an international student: "/non-uk-teachers/train-to-teach-in-england-as-an-international-student"
+    Teacher pay and benefits in England : "/is-teaching-right-for-me/teacher-pay-and-benefits"
+    A day in the life of a teacher in England : "/a-day-in-the-life-of-a-teacher"
+promo_content:
+navigation: 20.60
+navigation_title: Find a teaching job as a non-UK citizen
+navigation_description: How to successfully apply for your first teaching job.
+date: "2021-05-27"
+image: "static/images/content/hero-images/0002.jpg"
+backlink: "../../"
+keywords:
+  - International
+  - Teaching
+  - Education
+  - Schools
+  - Teacher
+  - Overseas
+  - Foreign
+  - Non-UK
+  - Visa
+  - Skilled Worker visa
+  - Immigration
+  - Student Visa
+  - Graduate Visa
+  - Youth Mobility Scheme
+  - Family Visa
+  - UK Ancestry Visa
+  - Refugees
+  - Settlement Scheme
+  
+---
+
+English [qualified teacher status](/train-to-be-a-teacher/what-is-qts) allows you to teach in any type of school in England.
+
+Depending on your immigration status, you may have to [apply for a new visa](/non-uk-teachers/visas-for-non-uk-teachers) (usually a Skilled worker visa) to work as a teacher in the UK. 
+
+## Apply for your teaching job  
+
+To find a teaching job, you must apply to individual schools, not the Department for Education. If you need a Skilled Worker visa, you must find a school that can sponsor your visa. 
+
+You can search for a teaching job through the government’s official [Teaching Vacancies service](https://teaching-vacancies.service.gov.uk/). Filter your search by ‘visa sponsorship’ to find jobs that can sponsor visas. Set up job alerts so you don’t miss out on new roles. 
+
+Most teaching vacancies are advertised in March, April or the first half of May, with jobs usually starting at the beginning of the school year in September.  
```

Comment:
not strictly true - schools can (and do) advertise roles year round, it's just the peak time is May because thats the deadline for teachers handing in their notice for the following year as they have to give their notice a term in advance!
@Sarah-DfE